python/lookerstudio/README.md

```diff
@@ -1,5 +1,32 @@
 # Marketing Analytics Jumpstart Looker Studio Dashboard
 
+## Prerequisites
+This Looker Studio dashboard relies on specific BigQuery tables that should be present in your project. These tables are created during the deployment of the Marketing Analytics Jumpstart and by the data processing pipelines of the solution.
+Before deploying the dashboard, make sure the pre-requisite tables exist. If tables are missing, ensure the corresponding pipelines have run successfully.
+
+| Table |	Dataset | Source Process | Troubleshooting Link |
```

Comment:
Hey @kingman for all the links to the google cloud console, you can remove the `project-id` substring, it will be automatically fetched from the latest project the person has selected.

Reply:
fixed